packages/interbit-ui-components/src/components/BlockExplorer/index.js

```diff
@@ -59,14 +59,17 @@ export default class BlockExplorer extends Component {
     const {
       selectedChain,
       showRawData,
-      doSetSelectedBlockHash,
+      doSelectBlock,
       selectedBlockHash
     } = this.props
 
     const blocks = selectedChain.blocks
     const treeData = {
-      ...selectedChain.state,
-      interbit: { ...selectedChain.interbit, blocks: selectedChain.blocks }
+      state: {
+        ...selectedChain.state,
+        interbit: selectedChain.interbit
```

Comment:
Here again, this spread can be simplified.

Reply:
It could, but again this is intentional.